### PR TITLE
Add information about OpenJ9 MXBeans

### PR DESCRIPTION
**Pull request by SueChaplain**
Add new topic and change links from old content in
IBM Knowledge Center.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Docs staged: https://eclipse.github.io/openj9-docs-staging/461/interface_lang_management/

File: docs/interface_lang_management.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+| MXBean                    | Description                                                                                                  |
+|---------------------------|--------------------------------------------------------------------------------------------------------------|
+| `GarbageCollectorMXBean`    | Discovers Garbage Collection (GC) operations (collection times, compactions, heap memory usage, and freed memory). |
+| `MemoryMXBean`              | Discovers memory usage (minimum and maximum heap sizes, and shared class cache sizes).               |
+| `MemoryPoolMXBean`          | Discovers memory pool usage for specific GC policies.                                                         |
+| `ThreadMXBean`              | Discovers information about native thread IDs.                                                                |
+| `JvmCpuMonitorMXBean`       | Discovers CPU consumption by category (GC, JIT or Other threads).                                             |
+| `ProcessorMXBean`           | Discovers and limits the CPUs that a process has access to.                                                  |
```
Comment (pshipton) [inline docs/interface_lang_management.md]: This one shouldn't be in the list. It's not a properly installed MXBean. It shouldn't be in the IBM doc either, I guess we forgot to remove it in the last update.

File: docs/interface_lang_management.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Language management interface
+
+Eclipse OpenJ9 provides MXBean extensions to the standard `java.lang.management` API, which can be used to monitor and manage the Java@reg; virtual machine.
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/interface_lang_management.md]: Must have been having a Christmas moment!

File: docs/interface_lang_management.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+These extensions provide access to information about the state of the OpenJ9 VM and the environment in which it is running. The following
+table lists the MXBeans and describes the monitoring or management capability:
+
+
+| MXBean                    | Description                                                                                                  |
+|---------------------------|--------------------------------------------------------------------------------------------------------------|
+| `GarbageCollectorMXBean`    | Discovers Garbage Collection (GC) operations (collection times, compactions, heap memory usage, and freed memory). |
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/interface_lang_management.md]: I was trying to group them by the resources they were targeting, but I've no particular preference. Will reorder.

File: docs/interface_lang_management.md (hunk 1 above)
Comment (SueChaplain) [reply docs/interface_lang_management.md]: Removed

File: docs/interface_lang_management.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+| `MemoryMXBean`              | Discovers memory usage (minimum and maximum heap sizes, and shared class cache sizes).               |
+| `MemoryPoolMXBean`          | Discovers memory pool usage for specific GC policies.                                                         |
+| `ThreadMXBean`              | Discovers information about native thread IDs.                                                                |
+| `JvmCpuMonitorMXBean`       | Discovers CPU consumption by category (GC, JIT or Other threads).                                             |
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/interface_lang_management.md]: Fixed

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/interface_lang_management.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Language management interface
+
+Eclipse OpenJ9 provides MXBean extensions to the standard `java.lang.management` API, which can be used to monitor and manage the Java@trade; virtual machine.
```
Comment (pshipton) [inline docs/interface_lang_management.md]: Still `@trade` when it should be `&trade`

File: docs/interface_lang_management.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+| `ThreadMXBean`              | Discovers information about native thread IDs.                                                                |
+| `UnixOperatingSystemMXBean` | Discovers information for Unix operating systems (memory, file descriptors, processors, processor usage, and hardware)|
+
+| `com.ibm.virtualization.management` MXBean | Description                                                                                                  |
+|---------------------------|--------------------------------------------------------------------------------------------------------------|
+| `GuestOSMXBean`             | Discovers CPU and memory statistics of a virtual machine or logical partition as seen by the Hypervisor.       |
+| `HypervisorMXBean`          | Discovers whether the operating system is running on a hypervisor and provides information about the hypervisor.|
+
+
+| `openj9.lang.management` MXBean | Description                                                                                                  |
```
Comment (pshipton) [inline docs/interface_lang_management.md]: Can the header be forced onto two lines like the others above?

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/interface_lang_management.md (hunk 5 above)
Comment (SueChaplain) [reply docs/interface_lang_management.md]: Urgh

File: docs/interface_lang_management.md (hunk 6 above)
Comment (SueChaplain) [reply docs/interface_lang_management.md]: Tables are quite restrictive in markdown. I lifted the package names out as headers to the tables - hopefully make it look a bit better. Staging now.

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Pull request merged**